routes/game.rb

```diff
@@ -115,11 +115,16 @@ class Api
               end
             end
 
+            other_users = users.reject { |u| u.id == user&.id }
             type, user_ids, force =
               if action['type'] == 'message'
-                pinged = users.select do |user|
-                  action['message'].include?("@#{user.name}")
-                end
+                pinged = if %w[@all @everyone @here].include?(action['message'])
```

Comment:
what happens if there's a user named these?, also we probably only need one

Reply:
ah makes sense, I'd probably just use `@all` then
I created a username to squat on `@all`